data/beaconrestapi/src/test/java/tech/pegasys/teku/beaconrestapi/BeaconRestApiTypesTest.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Consensys Software Inc., 2023
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.beaconrestapi;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EnumSource;
+import tech.pegasys.teku.beaconrestapi.BeaconRestApiTypes.BroadcastValidationParameter;
+
+public class BeaconRestApiTypesTest {
+
+  @ParameterizedTest
+  @EnumSource(BroadcastValidationParameter.class)
+  void broadcastValidationParameter_anInternalVersionMustExist(
+      final BroadcastValidationParameter parameter) {
+    parameter.toInternal();
```

Comment:
A bit confusing, what is it for? Maybe wrap in `assertNotThrow(() ->`?

Reply:
ah yes, would me more clear